assets/css/2-base/_header.sass

```diff
@@ -3,6 +3,9 @@ header
 
   .header-image
     height: 30rem
+    display: block
+    pointer-events: none
+    cursor: default
```

Comment:
I'm not sure if this is necessary.
Might be better to leave this out so that user has a consistent experience across different screen sizes.

Reply:
I‘m not sure about that. I wouldn’t expect a header-image to link to the landing page, except the header image is simply the logo.

Reply:
And on desktop you can go to the landing page with just one click

Reply:
@gruselhaus what do you think about it?

Reply:
> And on desktop you can go to the landing page with just one click

To be honest I don't think this feature is necessary at all. On mobile, to navigate anywhere else user already have to click on the Menu button anyway, why would the home button be treated differently, and also creating more complexity for user: 
- Inconsistent behavior for the same visual component (the logo image)
- Also create potential issue where user miss click the menu button and get redirected
- If the menu layout is logical, the user shouldn't have to navigate back to the Home page

Reply:
I am with @phucduong86